pkg/lib/cockpit-components-table.jsx

```diff
@@ -151,7 +151,10 @@ export const ListingTable = ({
                             </Text>
                         </TextContent>
                     </EmptyStateBody>
-                    {actions.length > 0 ? <EmptyStateSecondaryActions>{actions}</EmptyStateSecondaryActions> : null}
+                    {actions.length > 0 &&
+                    <EmptyStateFooter>
+                        <EmptyStateActions>{actions}</EmptyStateActions>
+                    </EmptyStateFooter>}
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230425-151143-725ccf60-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)